.buildkite/pipeline.yml

```diff
@@ -150,7 +150,7 @@ steps:
         artifact_paths: "sphere_baroclinic_wave_rhoe_equilmoist/*"
 
       - label: ":computer: baroclinic wave (ρe) equilmoist radiation"
-        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --moist equil --microphy 0M --rad clearsky --job_id sphere_baroclinic_wave_rhoe_equilmoist_radiation --dt 300"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --moist equil --microphy 0M --rad clearsky --job_id sphere_baroclinic_wave_rhoe_equilmoist_radiation --dt 200 --t_end 432000"
```

Comment:
@szy21, it looks like this doesn't end up running to the time when we create the output plots (see [here](https://github.com/CliMA/ClimaAtmos.jl/blob/74727ebf7141c0ecced5c559489a77c95a030778/examples/hybrid/define_post_processing.jl#L141-L146)).

Reply:
Yes, I'll just change it to postprocessing rather than paperplots by changing is_baro_wave. The baroclinic_wave_with_radiation test case is not a standard baroclinic wave anyway.